src/util/get-files.ts

```diff
@@ -54,17 +76,17 @@ const walkSync = async (dir, path, filelist = [], { output } = {}) => {
  *  - `output` {Object} "output" helper object
  * @returns {Array} the expanded list of whitelisted files.
  */
-const getFilesInWhitelist = async function(whitelist, path, { output } = {}) {
-  const { debug } = output;
-  const files = [];
+const getFilesInWhitelist = async function(whitelist: string[], path: string, opts: FilesInWhitelistOptions) {
+  const { debug } = opts.output;
+  const files: string[] = [];
 
   await Promise.all(
-    whitelist.map(async file => {
+    whitelist.map(async (file: string) => {
       file = asAbsolute(file, path);
       try {
         const file_stat = await fs.promises.stat(file);
         if (file_stat.isDirectory()) {
-          const dir_files = await walkSync(file, path, [], { output });
+          const dir_files = await walkSync(file, path, [], opts);
```

Comment:
Thoughts on passing `opts` here instead of creating a new object each time? I noticed they all do this.

Reply:
Seems like a good idea to me.